### PR TITLE
Hw cyber apocalypse

### PR DESCRIPTION
**Pull request by ShiroUsagi-san**
adding more cyber apocalypse writeups

### CODE REVIEW

File: book.toml [hunk 1]
```diff
@@ -8,9 +8,9 @@ multilingual = false
 src = "src"
 title = "CTF Writeups"
 create-missing = false
-
 [output.html.fold]
 enable = true
-
```
Comment (lightdiscord) [inline book.toml:14]: Is it necessary to remove empty lines ?

File: src/cyber-apocalypse-2021/hardware/compromised/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+![dump_logic](https://i.imgur.com/OQz980v.png)
+
+This is a I²C communication with channel 0 being SDA and channel 1 SCL, the clock.
+We can add a I²C analyzer on logic and extract the analysis to `dump.txt`.
+```
+{{#include dump.txt:1:10}}
+```
+Then we will use a python script to find the flag. 
+```python
+{{#include script.py:loading}}
```
Comment (lightdiscord) [inline src/cyber-apocalypse-2021/hardware/compromised/README.md]: The `loading` anchor is missing from `script.py`

File: src/cyber-apocalypse-2021/hardware/serial_logs/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Hw - Serial logs
```
Comment (lightdiscord) [inline src/cyber-apocalypse-2021/hardware/serial_logs/README.md:1]: Forgot to add in SUMMARY

File: src/cyber-apocalypse-2021/hardware/serial_logs/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+For this challenge, we have a Saleae Logic project export. Giving the name of the challenge, and the channel 1, we will use the `serial Async` with the default settings:
+![dump_img_serial](https://i.imgur.com/olqr2nt.png)
+
+Somes logs appear then in the console:
```
Comment (clemg) [inline src/cyber-apocalypse-2021/hardware/serial_logs/README.md]: Somes ?

File: src/cyber-apocalypse-2021/hardware/serial_logs/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+```
+[...more logs...]
+[LOG] Connection from 6edec472e9754574d91f460e170b825bacee5f121b73805dffa4f2a5a7d23d7f
+[LOG] Connection from 316636cf0500c22f97fa261585b72a48c4625aca7868f0f6ee253937620ac15c
+[LOG] Connection from 4b1186d29d6b97f290844407273044e5202ddf8922163077b4a82615fdb22376
+[LOG] Connection from 4b1186d29d6b97f290844407273044e5202ddf8922163077b4a82615fdb22376
+[LOG] Connection from 4b1186d29d6b97f290844407273044e5202ddf8922163077b4a82615fdb22376
+[ERR] Noise detected in channel. Swithcing baud to backup value
```
Comment (clemg) [inline src/cyber-apocalypse-2021/hardware/serial_logs/README.md:13]: Should this be "switching" ?

File: src/cyber-apocalypse-2021/hardware/serial_logs/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+```
+And after this error, there is garbage on the console output. This last message learn us that the channel baud has been changed from `115200` to some unknown value. After trying some common default values without success, we will try to figure out the baud by analysing the capture.
+For this, we first take a charactere than could be found in the damaged communication.
+![baud_calculation](https://i.imgur.com/dqrisQa.png)
+
+So the timing for the first bit of communication is `8.48μs`.
+We then compute \\(\frac{1}{8.48\times 10^{-6}\\) to find the baud (which is roughly like a frequency)
```
Comment (lightdiscord) [inline src/cyber-apocalypse-2021/hardware/serial_logs/README.md]: Missing closing curly-brace for `\frac`

**Review by lightdiscord — CHANGES_REQUESTED** [review]

File: src/cyber-apocalypse-2021/hardware/serial_logs/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+```
+[...more logs...]
+[LOG] Connection from 6edec472e9754574d91f460e170b825bacee5f121b73805dffa4f2a5a7d23d7f
+[LOG] Connection from 316636cf0500c22f97fa261585b72a48c4625aca7868f0f6ee253937620ac15c
+[LOG] Connection from 4b1186d29d6b97f290844407273044e5202ddf8922163077b4a82615fdb22376
+[LOG] Connection from 4b1186d29d6b97f290844407273044e5202ddf8922163077b4a82615fdb22376
+[LOG] Connection from 4b1186d29d6b97f290844407273044e5202ddf8922163077b4a82615fdb22376
+[ERR] Noise detected in channel. Swithcing baud to backup value
+```
+And after this error, there is garbage on the console output. This last message learn us that the channel baud has been changed from `115200` to some unknown value. After trying some common default values without success, we will try to figure out the baud by analysing the capture.
```
Comment (clemg) [inline src/cyber-apocalypse-2021/hardware/serial_logs/README.md]: Learn us does not exists, this should be "taught us".

File: src/cyber-apocalypse-2021/hardware/serial_logs/README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+For this, we first take a charactere than could be found in the damaged communication.
```
Comment (clemg) [inline src/cyber-apocalypse-2021/hardware/serial_logs/README.md]: "That" could be found*

**Review by clemg — CHANGES_REQUESTED** [review]
Clément, official strike corrector of the French national team

**Review by clemg — APPROVED** [review]

**Review by lightdiscord — APPROVED** [review]

**Pull request merged**